docs/books/user-guide/routing.adoc

```diff
@@ -551,6 +551,87 @@ autoLink {
 For information about additional attributes, see link:{qdrouterdConfManPageUrl}#_autolink[autoLink] in the `qdrouterd.conf` man page.
 --
 
+=== Handling Undeliverable Messages for an Address
+
+You handle undeliverable messages for an address by configuring _fallback destinations_.
+A fallback destination is a queue on a broker that stores messages that are not directly routable to any consumers.
```

Comment:
This is probably the most common use for fallback-destination.  However, there is no reason that the fallback destination must be a queue on a broker.